runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -127,4 +127,15 @@
   @Default.Boolean(false)
   Boolean getRetainExternalizedCheckpointsOnCancellation();
   void setRetainExternalizedCheckpointsOnCancellation(Boolean retainOnCancellation);
+
+  @Description("Max elements size of Bundle trigger.")
+  @Default.Long(1000)
+  Long getMaxBundleSize();
+  void setMaxBundleSize(Long size);
+
+  @Description("Max time mills of Bundle trigger.")
+  @Default.Long(1000)
+  Long getMaxBundleTime();
```

Comment:
And for this one: `The maximum time to wait before finalising a bundle (in milliseconds).`